pd/src/state.rs

```diff
@@ -403,6 +415,8 @@ ON CONFLICT (id) DO UPDATE SET data = $1
 
     pub async fn rate_data(&self, epoch_index: u64) -> Result<Vec<RateData>> {
         let mut conn = self.pool.acquire().await?;
+        // TODO: This query needs to be updated to select the *most recent* rate data
+        // to the given epoch
```

Comment:
Can we make an issue for this TODO?

Reply:
https://github.com/penumbra-zone/penumbra/issues/402